src/pageObject.js

```diff
@@ -1,15 +1,38 @@
 'use strict';
 
 class Header {
-
+  get logo() {
+    return 'Conduit logo';
+  }
 };
 
 class PageObject {
+  constructor(url) {
+    this.header = new Header();
+    this.url = 'http://test.com';
+  }
 
+  clickOnLogo() {
+    return `Click on the ${this.header.logo}`;
+  }
 };
 
 class ArticlePage extends PageObject {
+  constructor(url, slug) {
+    super(url);
+    this.slug = slug;
+    this.url = url + slug;
+  }
```

Comment:
you can separate parts of the code with an empty line, so the code will be easier to read

Reply:
Done